spec/features/proposal/access_permissions/no_permissions_for_draft_mmt_spec.rb

```diff
@@ -158,3 +158,38 @@
     end
   end
 end
+
+describe 'No Permissions for Non-NASA Workflow in MMT', js: true do
```

Comment:
since this is testing what is happening in MMT Proper, I think you should make a folder for 'Manage Proposals', and move all these new permissions tests into their own files.